Description: Fix tips-of-the-day window with GCC 4.7
 When making the text file tips-of-the-day, GnuCash expects 'gcc -E' to
 preserve at least one of the whitespace lines between entries. However, this
 relies on behavior of 'gcc -E' that isn't actually part of the spec, and is
 a historical accident. And it changed in gcc-4.7, such that all the
 whitespace is removed.
 .
 Work around this by explicitly adding a newline in the sed expression.
Author: Bill Nottingham <notting@redhat.com>
Forwarded: http://lists.gnucash.org/pipermail/gnucash-devel/2012-June/034023.html
Applied-Upstream: 2.4.11, commit: 22247
Last-Update: 2012-06-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -52,7 +52,7 @@
 tip_of_the_day.list: tip_of_the_day.list.in Makefile
 	${CC} -E -P -x c -D'N_(x)=x' -o $@.tmp $<
 	cat -s $@.tmp | ${SED} -e 's/^ *"//' \
-	                       -e 's/" *$$//' \
+	                       -e 's/" *$$/\n/' \
 	                       -e 's/"* *[|] */|/' \
                            -e 's:@-GNUCASH_LATEST_STABLE_SERIES-@:${GNUCASH_LATEST_STABLE_SERIES}:g' > $@
 	rm -f $@.tmp
